.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,9 +5,10 @@
 
 ## Description of Changes
 
+<!--- Remember that an image/animation is word a thousand words! --->
```

Comment:
I assume you mean *worth* a thousand words...also, it might have been best to include this directly above the relevant bullet, rather than two unrelated ones.

Reply:
Yes worth 🤣🤣🤣 will issue a fix?